src/control/destination/mod.rs

```diff
@@ -180,18 +182,21 @@ impl Metadata {
             labels: IndexMap::default(),
             protocol_hint: ProtocolHint::Unknown,
             identity: None,
+            weight: 10_000,
```

Comment:
Could we parameterize the `10_000`? I was little confused why the number was chosen in [here](https://github.com/linkerd/linkerd2-proxy/pull/230/files#diff-259de4b6a0f288cb0b854d2c3e6062ffR73).

Reply:
we can constify it... but this just equals "1.0" -- the grpc API sends weight as a uint32. We represent 1.0 as 10_000 so the smallest value we can represent is 0.0001.  Practically, it doesn't matter at all because we'll _always_ set a value from the API when balancing is used (once eliza's changes are in, anyway)

Reply:
I've added a comment and consolidated the logic in that file.